src/components/Entry/DomainArchitectures/Options/index.tsx

```diff
@@ -73,11 +74,14 @@ const IDAOptions = ({
           <div className={css('menu-grid')}>
             <label htmlFor="json">JSON</label>
             <AllIDADownload count={count} fileType="json" />
-            <label htmlFor="json">TSV</label>
+            <label htmlFor="json">JSON</label>
             <AllIDADownload count={count} fileType="tsv" />
             <label htmlFor="api">API</label>
             {api && (
-              <APIViewButton url={getAPIURL(api, entryLocation, search)} />
+              <ExternalExportButton
+                type={'api'}
+                url={getAPIURL(api, entryLocation, search)}
```

Comment:
Make sure that `getAPIURL` returns the URL of the public API, not the one used by the website (`wwwapi`):

```ts
const getAPIURL = (
  { protocol, hostname, port, root }: ParsedURLServer,
  entryLocation?: EndpointLocation,
  search?: InterProLocationSearch,
) => {
  const description: InterProPartialDescription = {
    main: { key: 'entry' },
    entry: {
      ...(entryLocation || {}),
      detail: undefined,
    },
  };
  return toPublicAPI(
    format({
      protocol,
      hostname,
      port,
      pathname: root + descriptionToPath(description),
      query: entryLocation?.accession ? { ida: '' } : search,
    })
  );
};
```